### DISCUSSION TITLE
Join & outline => Input prefix & suffix

### DISCUSSION THREAD

**Discussion by jycouet** [saadeghi/daisyui#2751] [Ideas / feature requests]
I would like to use `join` to create Input with prefix & suffix, something like this:

https://github.com/saadeghi/daisyui/assets/5312607/8220264d-bbbd-4fa1-8ddc-0dce73887f5e

To do so, I removed border, but didn't manage to have the "full" outline.
I used something like this:
https://play.tailwindcss.com/l0nXk0rn4s
```html
<div class="p-4">
  <div class="join flex"> <!-- with a global `outline-2`? -->
    <div>
      <div>
        <input type="number" class=" input join-item input-bordered border-r-0 text-right" placeholder="Value..." value="1234.23" />
      </div>
    </div>
    <div>
      <div>
        <div class="input join-item input-bordered grid items-center border-l-0">€</div>
      </div>
    </div>
  </div>
</div>
```

I'm I missing something? Could I manage it on my side? Or DaisyUI could provide something? :)

**Comment by saadeghi** [comment]
I would do something like this:  
https://play.tailwindcss.com/8hEY0E6bXI
```html
<label class="input input-bordered inline-flex items-center">
  <input type="number" class="h-full text-end" placeholder="Value..." value="1234.23" />
  €
</label>
```

The `input` class is not necessarily for `<input>` element. It just gives the style, so we can use it for the parent instead since we want a `€` next to the `<input>`

**Reply by jycouet** [reply]
OMG, that's just too easy! Thx a lot
